tests/gtests/graph/unit/backend/elyzor/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+#===============================================================================
```

Comment:
this was copied from the [tests/gtests/graph/unit/backend/graph_compiler/CMakeLists.txt](https://github.com/kurapov-peter/oneDNN/pull/3/files#diff-fd0df981389b4dce46a42ca4de1b6ea7836a054376560d4407d00338d4a12daf) with the respective renamings